docs/reference/modules/cluster/shards_allocation.asciidoc

```diff
@@ -57,17 +57,18 @@ one of the active allocation ids in the cluster state.
 [[shards-rebalancing-settings]]
 ==== Shard rebalancing settings
 
-A cluster is _balanced_ when it has an equal number of shards on each node
-without having a concentration of shards from any index on any node. {es} runs
-an automatic process called _rebalancing_ which moves shards between the nodes
-in your cluster to improve its balance. Rebalancing obeys all other shard
-allocation rules such as <<cluster-shard-allocation-filtering,allocation
-filtering>> and <<forced-awareness,forced awareness>> which may prevent it from
-completely balancing the cluster. In that case, rebalancing strives to achieve
-the most balanced cluster possible within the rules you have configured. If you
-are using <<data-tiers,data tiers>> then {es} automatically applies allocation
-filtering rules to place each shard within the appropriate tier. These rules
-mean that the balancer works independently within each tier.
+A cluster is _balanced_ when it has an equal number of shards on each node, with
+all nodes needing equal resources, without having a concentration of shards from
```

Comment:
Should we give an example for resources to be more specific here (eg disk usage and may be ingest load)?

Reply:
I'm ok with being vague here - we go into more detail lower down, and the general principle really applies to all resources even if we don't consider most of them today.